### PR TITLE
added white list

### PR DESCRIPTION
**Pull request by ytwang0320**

### CODE REVIEW

File: tools/ipvsadm/ipvsadm.c [hunk 1]
```diff
@@ -196,12 +202,14 @@ static const char* optnames[] = {
...
 	"local-address" ,
 	"blklst-address",
+	"whtlst-address",
 	"synproxy" ,
 	"ifname" ,
 	"sockpair" ,
 	"hash-target",
 	"cpu",
 	"expire-quiescent",
+	"wlst",
```
Comment (ywc689) [inline tools/ipvsadm/ipvsadm.c:212]: What's the purpose of "wlst" option?

**Review by ywc689 — CHANGES_REQUESTED** [review]

**Comment by haosdent** [comment]
Cool!

**Review by ywc689 — APPROVED** [review]

File: src/ipvs/ip_vs_whtlst.c [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+    cf.af    = af;
+    cf.vport = vport;
+    cf.proto = proto;
+
+    /*del whtlst ip on master lcores*/
+    err = dp_vs_whtlst_del_lcore(af, proto, vaddr, vport, whtlst);
+    if (err) {
+        RTE_LOG(INFO, SERVICE, "[%s] fail to del whtlst ip\n", __func__);
+        return err;
+    }
```
Comment (ywc689) [inline src/ipvs/ip_vs_whtlst.c:214]: Don't log for EDPVS_NOTEXIST.

```
if (err && EDPVS_NOTEXIST) {
    RTE_LOG(INFO, SERVICE, "[%s] fail to del whtlst ip: %s\n", __func__, dpvs_strerror(err));
    return err;
}
```

File: src/ipvs/ip_vs_whtlst.c [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+    cf.af    = af;
+    cf.vport = vport;
+    cf.proto = proto;
+
+    /*set whtlst ip on master lcore*/
+    err = dp_vs_whtlst_add_lcore(af, proto, vaddr, vport, whtlst);
+    if (err) {
+        RTE_LOG(INFO, SERVICE, "[%s] fail to set whtlst ip\n", __func__);
+        return err;
+    }
```
Comment (ywc689) [inline src/ipvs/ip_vs_whtlst.c:171]: Don't log for EDPVS_EXIST.

```
if (err && err != EDPVS_EXIST) {
    RTE_LOG(INFO, SERVICE, "[%s] fail to set whtlst ip: %s\n", __func__, dpvs_strerror(err));
    return err;
}
```

**Review by ywc689 — CHANGES_REQUESTED** [review]
Please adjust logs.

**Review by ywc689 — APPROVED** [review]

**Pull request merged**